docs/cql/cql-extensions.md

```diff
@@ -133,21 +133,34 @@ token ranges.
 
 ## Synchronous materialized views
 
-Materialized view updates can be applied synchronously (with errors propagated
-back to the user) or asynchronously, in the background. Historically, in order
-to use synchronous updates, the materialized view had to be local,
-which could be achieved e.g. by using the same partition key definition
-as the one present in the base table.
-Scylla also allows explicitly marking the view as synchronous, which forces
-all its view updates to be updated synchronously. Such views tend to reduce
-observed availability of the base table, because a base table write would only
-succeed if all synchronous view updates also succeed. On the other hand,
-failed view updates would be detected immediately, and appropriate action
-can be taken (e.g. pruning the materialized view, as mentioned in the paragraph
-above).
-
-In order to mark a materialized view as synchronous, one can use the following
-syntax:
+Usually, when a table with materialized views is updated, the update to the
+views happen _asynchronously_, i.e., in the background. This means that
+the user cannot know when the view updates have all finished - or even be
+sure that they succeeded.
+
+In Scyla (but not in Cassandra), in some special cases the view updates
+are done _synchronously_ - i.e., the user's write returns only after the
+views were updated. View updates are synchronous when the materialized view
+replica is on the same node as the base-table replica. For example, if the
+base-table and the view have the same partition key.
```

Comment:
Can you change the spelling to make it consistent on the page? It should be "base table".

Reply:
Sure, but I guess you mean "base table" as a noun, "base-table" as a compound adjective (e.g., "base-table replica"). That is the rule I was taught...
Anyway you're write that in the last sentence it should be "base table".

Reply:
Yes, I meant the noun. It looks great now.